src/gui/systray.cpp

```diff
@@ -389,8 +389,9 @@ QRect Systray::taskbarGeometry() const
     return tbRect;
 #elif defined(Q_OS_MACOS)
     // Finder bar is always 22px height on macOS (when treating as effective pixels)
-    auto screenWidth = currentScreenRect().width();
-    return {0, 0, screenWidth, 22};
+    const auto screenWidth = currentScreenRect().width();
+    const auto statusBarHeight = (int)OCC::statusBarThickness();
```

Comment:
@claucambra Can you use `C++` style cast (`static_cast<int>())`) instead of `C-style`?

Reply:
good catch
sorry for missing this

Reply:
Got it, changed the cast, thanks